json/src/spec/builtin.rs

```diff
@@ -16,50 +16,75 @@
 
 //! Spec builtin deserialization.
 
+use std::collections::BTreeMap;
+
 use crate::uint::Uint;
 use serde::Deserialize;
 
-
 /// Linear pricing.
 #[derive(Debug, PartialEq, Deserialize, Clone)]
 #[serde(deny_unknown_fields)]
 pub struct Linear {
 	/// Base price.
-	pub base: usize,
+	pub base: u64,
 	/// Price for word.
-	pub word: usize,
+	pub word: u64,
 }
 
 /// Pricing for modular exponentiation.
 #[derive(Debug, PartialEq, Deserialize, Clone)]
 #[serde(deny_unknown_fields)]
 pub struct Modexp {
 	/// Price divisor.
-	pub divisor: usize,
+	pub divisor: u64,
 }
 
 /// Pricing for constant alt_bn128 operations (ECADD and ECMUL)
 #[derive(Debug, PartialEq, Deserialize, Clone)]
 #[serde(deny_unknown_fields)]
 pub struct AltBn128ConstOperations {
 	/// price
-	pub price: usize,
+	pub price: u64,
 	/// EIP 1108 transition price
-	pub eip1108_transition_price: usize,
+	// for backward compatibility
+	pub eip1108_transition_price: Option<u64>,
+}
+
+impl AltBn128ConstOperations {
+	fn clear_eip1108(&mut self) {
+		self.eip1108_transition_price = None;
```

Comment:
Not sure what is the purpose of this function. This crate should be used only for parsing